lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -339,6 +351,16 @@ def configure(conf)
             @resource.labels['cluster_name'] =
               cluster_name_from_kube_env(@kube_env)
             detect_cloudfunctions(attributes)
+          elsif attributes.include?('dataproc-cluster-uuid') &&
```

Comment:
Some unit tests for this as well? Should be similar to `test_one_container_log_metadata_from_plugin`.